src/psyclone/tests/domain/lfric/lfric_integer_builtins_test.py

```diff
@@ -44,11 +44,7 @@
 '''
 
 import os
-import pytest
-
```

Comment:
This file also has many calls to `code = str(psy.gen)` followed by asserts to the resulting code.

The psy.gen for builtins is now a single line: `parent.add(PSyIRGen(parent, self))` which in turn calls the `lower_to_language_level()`

It would be better to test these just with the "lower_to_language_level()" instead of the psy.gen because here we shouldn't be concerned about anything else that the psy-layer generates, just the built-in resulting code.

Would you mind doing the change? It is mostly deleting unneeded lines.

Reply:
Ok, I significantly pruned down this test. I left the full-blown checks with distributed memory and annexed DoFs in the first built-in, though, as we need to check for the correct looping limits and possible halo exchanges on "write" field. Otherwise I modified all other tests for a single run with distributed memory turned on (default for LFRic API), check for only the built-in code and compilation. Hopefully this is what you had in mind!